### PR TITLE
Update WebUI for metrics data changing to json string

### PR DESCRIPTION
**Pull request by lvybriage**
Update WebUI for metrics data changing to json string

### CODE REVIEW

File: src/webui/src/static/function.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ const getFinalResult = (final?: MetricDataRecord[]): number => {
     let acc;
     let showDefault = 0;
     if (final) {
-        acc = JSON.parse(final[final.length - 1].data);
+        acc = JSON5.parse(final[final.length - 1].data);
```
Comment (chicm-ms) [inline src/webui/src/static/function.ts]: Is it possible to call parseMetrics here?  there maybe more updates in parseMetrics later.

File: src/webui/src/static/function.ts [hunk 2]
```diff
@@ -182,8 +182,6 @@ function formatTimestamp(timestamp?: number, placeholder?: string): string {
 function parseMetrics(metricData: string): any {
     if (metricData.includes('NaN')) {
         return JSON5.parse(metricData)
-    } else {
-        return JSON.parse(metricData)
```
Comment (chicm-ms) [inline src/webui/src/static/function.ts]: why remove this?

File: src/webui/src/static/function.ts (hunk 1 above)
Comment (lvybriage) [reply src/webui/src/static/function.ts]: It's OK.

File: src/webui/src/static/function.ts (hunk 2 above)
Comment (lvybriage) [reply src/webui/src/static/function.ts]: I thought UI only use JSON5 from the teams message.
It's misleading, I'll change this back.

File: src/webui/src/static/function.ts [hunk 3]
```diff
@@ -61,7 +61,7 @@ const getFinalResult = (final?: MetricDataRecord[]): number => {
 const getFinal = (final?: MetricDataRecord[]): FinalType | undefined => {
     let showDefault: FinalType;
     if (final) {
-        showDefault = JSON.parse(final[final.length - 1].data);
+        showDefault = JSON5.parse(final[final.length - 1].data);
```
Comment (chicm-ms) [inline src/webui/src/static/function.ts]: Can this be replaced by parseMetrics?

Comment (lvybriage) [reply src/webui/src/static/function.ts]: OK.

**Pull request merged**